docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+---
+## DO NOT EDIT!
+# This file was provisioned by Terraform
```

Comment:
This comment can be removed, it's ok to make changes to this file specifically for this service if needed